routes/v1.js

```diff
@@ -986,13 +988,25 @@ api.declare({
     takenUntil:   run.takenUntil
   }, task.routes);
 
+  // Create temporary credentials for the task
+  let credentials = taskcluster.createTemporaryCredentials({
+    start:  new Date(Date.now() - 15 * 60 * 1000),
```

Comment:
Yeah... Yeah, I see no reason to be super strict...
It annoying with race conditions :)